### ISSUE TITLE
Localisation

### ISSUE THREAD

**Issue by tadhglewis** [tadhglewis/issue-status#26]
add messages to a localisation file so they can easily be translated

**Comment by ricardomaia** [comment]
Hi Tadhg! First all, I would like to say thank you for this project. It was an inspiration for my own work.
Localization is one of issues that impulse to make a sort of "fork" from "issue-status".

In my project I implement support to localization because I need this in my company. I'm military brazilian firefighter and the status page of my government corporation must be in portuguese.

Fell free to see my implementation at https://github.com/ricardomaia/statuspage.

Basically I create two functions and one json file:
 - https://github.com/ricardomaia/statuspage/blob/main/src/library/getLanguage.js (to detect browser language)
- https://github.com/ricardomaia/statuspage/blob/main/src/library/getMessage.js (to read tokens on json file)
- https://github.com/ricardomaia/statuspage/blob/main/src/messages.json

**Comment by tadhglewis** [comment]
@ricardomaia hi, great to hear i was able to help and inspire someone else.

my initial thoughts on localisation was most browsers translate web pages anyway so it wasn't really needed however i have changed my mind on this.

i was actually thinking of doing a similar thing to what you've done with the idea being all translations can be added to the main issue-status repo and available to anyone who forks/uses it, then on the status page itself have an optional (can hide with .env) language option/dropdown as well as a default option in the .env file

not sure when i'll get round to adding this due to work but it's definitely something i am interested in doing.

Let me know if you had any questions
